setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+from setuptools import find_packages
+
+setup(
+    name='tap-shiphero',
+    version='0.0.1',
+    classifiers=['Programming Language :: Python :: 3 :: Only'],
+    py_modules=['tap_shiphero'],
+    install_requires=[
+        'backoff==1.3.2',
+        'ratelimit==2.2.0',
+        'requests==2.20.1',
+        'pendulum==2.0.3',
```

Comment:
We'd prefer not to use pendulum at all.

Reply:
@KAllan357 Why? It's been used in a number of taps, I actually used to use arrow.py as my go-to datetime lib, but saw pendulum included in other Stitch code. Python's `datetime` and `timedelta` built-in modules have notoriously bad dev experience, hence arrow and pendulum's popularity.

Reply:
We chose not to use it because its actually pretty slow. We think the utilities available in singer-python's `utils` have been sufficient for our use cases and have been removing pendulum in preference of that.